src/kraken/std/python/pyproject.py

```diff
@@ -54,6 +54,35 @@ def save(self, path: Path | None = None) -> None:
     def get_poetry_sources(self) -> list[dict[str, Any]]:
         return list(self._poetry_section().setdefault("source", []))
 
+    def _find_dependencies_definitions(self, pyproject_section: Dict[str, Any], version: str) -> None:
+        """
+        Finds and updates the version of local dependencies listed in the
+        "dev-dependencies" and "dependencies" sections of the PyProject.toml file.
+        Args:
+            pyproject_section (Dict[str, Any]): A dictionary representing a section of the PyProject.toml file.
+            version (str): The version to update local dependencies with.
+        """
+        pattern = re.compile("^(dev-)?dependencies$")
+        for key, value in pyproject_section.items():
+            if pattern.match(key):
```

Comment:
```suggestion
            if key in ("dependencies", "dev-dependencies"):
```

This doesn't cover dependency groups, maybe add a comment to add the support in the future.

https://python-poetry.org/docs/master/managing-dependencies/#dependency-groups